xbmc/ServiceManager.cpp

```diff
@@ -23,9 +23,12 @@
 #include "utils/log.h"
 #include "interfaces/AnnouncementManager.h"
 #include "interfaces/generic/ScriptInvocationManager.h"
+#include "addons/binary/interfaces/api1/PVR/AddonCallbacksPVR.h"
 
 bool CServiceManager::Init1()
 {
+  V1::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
```

Comment:
Hi @FernetMenta I understand you about the hierarchy, but make for me no sense to have something like the follow in future:
```cpp
V1::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
V2::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
V3::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
V4::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
V5::KodiAPI::PVR::CAddonCallbacksPVR::EnableInterface(false);
```
better to have only one call on a part used by all.